### ISSUE TITLE
nvidia-dkms for Maxwell Cards

### ISSUE THREAD

**Issue by halimbo** [korvahannu/arch-nvidia-drivers-installation-guide#12]
I tried multiple times to get a working setup with the normal "nvidia" package.
The first time it actually worked was when i used the "nvidia-dkms" package.
You only mention "nvidia-340xx-dkms" and the other legacy ones but you still recommend "nvidia" for maxwell and newer.
My GPU is the GTX 750 Ti.
Im not 100% sure if "nvidia-dkms" was the reason i got it working but still quite.
Anyway it is still another way to try, even for the mainline "linux" kernel package. The arch wiki says "nvidia-dkms" is only needed for using any other kernel (eg "linux-zen").
"nvidia-dkms" is not mentioned in your guide and i think its not a bad idea to add it.

**Comment by korvahannu** [comment]
Hi! If you have time and interest you may check out the branch https://github.com/korvahannu/arch-nvidia-drivers-installation-guide/tree/driver-version-oq-545 if it solves the problem you faced. Someone else mentioned that both options A and B would be required for the regular "nvidia" -package to work in the new branch. I am still investigating this and will merge the branch when I get it resolved. I will make note to add a mention of the nvidia-dkms -package in the future. Thanks!

**Comment by halimbo** [comment]
Thx for your reply.
I saw it applies a slightly different syntax on the kernel line with "nvidia_drm modeset=1 fbdev=1".
My version was just "nvidia-drm.modeset=1". 
i wonder if the kernel just parses through this like theres no difference. 
The branch also doesnt recommend the "nvidia-dkms" package for maxwell cards. 
again i just followed your guide from start to finish with the exception of using "nvidia-dkms" instead of "nvidia". 
I also never explocitly installed "nouveau" so i guess i didnt have to blacklist it (not sure). At least i did not need to blacklist nouveau. Im getting full support for the gtx 750ti. (Testing with Age of Empires on Steam)
another thing i want to add that using "yay" is not essential for installing nvidia-dkms (i guess it is for the legacy packages though). 
Pacman alone is enough. I will formulate a short guide and post it as a fork or branch.

**Comment by halimbo** [comment]
Okay i have to apologize for the confusion. I did a fresh install, this time with the regular "nvidia" package and it also worked. I had to check if "nvidia-dkms" really made a difference.
still only using pacman and only adding "nvidia-drm.modeset=1". 
Still, its worth to mention that "nvidia-dkms" also works for the latest driver with mainline kernel.
And it is not mentioned in this guide.
And in the Archwiki it's only mentioned as an option for using any other kernel than the mainline one.
I am under the impression that mainline-kernel + regular nvidia package gave the lowest fps but thats something i would need to confirm and its not an issue at all with the installation itself.
thanks for making this guide, it really helped a lot.
